tests/test-backend-ops.cpp

```diff
@@ -3787,6 +3801,7 @@ struct test_rope : public test_case {
     bool ff;
     int v; // view (1 : non-contiguous a)
     bool forward;
+    bool inplace;
```

Comment:
You also need to update the function `vars` if you add a parameter.

Reply:
updated, thanks for the catch!